src/app/ec-card-preview/ec-card-preview.component.scss

```diff
@@ -0,0 +1,28 @@
+@use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
+
+.flashcard-edit {
+  text-align: center;
+}
+
+.preview-row{
+  display: flex;
+  flex: 1;
+  gap: 8px;
+}
+
+mat-icon {
```

Comment:
This causes the flip symbol on the other flashcards to change their position. Could you try to target only the menu icon in this component.

Reply:
I realize the entire component style was being imported into styles.scss. I moved the mixins into their own file so only the mixins get imported into styles.scss. I followed steps 1-4 from an [Angular Material Tutorial](https://material.angular.io/guide/theming-your-components).

I used the naming scheme (i.e., _componentname-theme.scss) used by google angular github.